### ISSUE TITLE
v7.18.0-iOS : can not login on iOS 13.3.1

### ISSUE THREAD

**Issue by pixiongames** [facebook/facebook-sdk-for-unity#383]
## Checklist

- [x ] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-sdk-for-unity/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Following the same issue that was closed, but on a newer iOS version and the latest Facebook SDK
https://github.com/facebook/facebook-sdk-for-unity/issues/333

[https://camo.githubusercontent.com/88827e557224dff14ef248daa5a475686446820b/68747470733a2f2f7265736f757263652d313235353730333538302e636f732e61702d7368616e676861692e6d7971636c6f75642e636f6d2f556e6974792f636f6d70616e792f6c6579692f696f735f66616365626f6f6b5f6c6f67696e5f6661696c65642e504e47](url)

- Unity Editor Version: `2019.2.6f1`
- Unity SDK Version: `7.18.0`
- Installation Platform & Verison: `[iOS|Android|WebGL]` version `iOS 13.3.1`

## Goals

We want to be able to Login via Facebook

## Expected Results

Facebook Login should succeed or allow us to cancel the operation if an error happens

## Actual Results

We are stuck in a window that doesn't allow us to proceed any further

## Steps to Reproduce

Try to login to Facebook on a fresh application install

## Code Samples & Details

N/A

**Comment by PeanutPro** [comment]
That's what's happening to us as well. The open or cancel button in the middle don't respond.
![image](https://user-images.githubusercontent.com/60428945/73366779-6c778d00-42a6-11ea-9bbe-8abffa8695f2.png)

**Comment by h3902340** [comment]
Yes, I cannot login via Facebook either. I am using facebook sdk for Unity 7.16.1 and Unity 2019.2.20f1, and I am testing an iPhone X, its os version is 13.3.
When I clicked login, Facebook app didn't launch. When the Facebook app wasn't installed, the built-in login browser didn't launch, either.

**Comment by 3goandrearossini** [comment]
Any news on this?
We've the same issue with Unity 2017.4.36, XCode 11.3.1, Facebook SDK 7.18.1 and iOS 13.3.1

**Comment by 3goandrearossini** [comment]
A small update:

I've achieved Facebook Login with Unity 2017.4.36, Facebook SDK 7.18.1 and iOS 13.3.1.
I've done this steps (no code required):

Launch the login flow normally.
The browser is opened. When it asks to open facebook app I click cancel (you can also click open, but in this case you will be redirected to an app page where you will click "Consent" and it will returns you to the browser page)
On Browser I navigate to the facebook login page. Log In here.
Close the app, relaunch and restart the flow.
From the browser page there is now a "Confirm" button that triggers the correct login callback to the app.
This does not fix the problem that the login flow doesn't work with the Facebook App. Is there a way to force Unity to launch the Facebook App instead of browser?

**Comment by gpiffaretti** [comment]
This is driving me crazy, I'm not being able to release because of this bug and I know what else to do :(

**Comment by yurykorzun** [comment]
Could be related to this https://github.com/facebook/facebook-sdk-for-unity/pull/216

**Comment by yurykorzun** [comment]
https://github.com/facebook/facebook-ios-sdk/issues/1067

**Comment by vincent0y23** [comment]
act the same as @PeanutPro 
IOS Version:13+, UnityVersion:2018.4.11f, fbSdkVersion:7.19.2

If it meets all 3 follow situations，it can reappear:
1 Iphone installed Facebook without login
2 The rotation of screen is locked
3 The game is horizontal.

Steps to Reproduce:
restart iphone->Enter Game->Login(will open browser in game and auto popup confirm-dialog)->click open(will open facebook app)->click return to game(leftup side)

!!!From now on your login will stuct in popup confirm-dialog step.Only the top left return button is clickable,any other component cannot be clicked.
Restarting the game can't solve the problem.
The same as other horizontal screen games.

There are 3 bad solutions:
1, Resert iphone(It's back to the beginning).
2, Open Facebook sdk in Vertical screen game(You will always avoid this bug before restarting your phone).
3, Unlock the rotation of iphone, rotate the iphone(You will always avoid this bug before restarting your phone).

Login confirm-dialog is showed by safariViewController. I don't know  what causes the bug. Is the login web page or the way to use safariViewController?

Is there any suggestions?THX~!

**Comment by cdelvalleJC** [comment]
I fixed this issue by unticking the "Render extra frame on pause" option in the iOS Player Settings area.  https://github.com/facebook/facebook-sdk-for-unity/issues/436
